api/server/sdk/cloud_backup.go

```diff
@@ -354,10 +354,12 @@ func (s *CloudBackupServer) StateChange(
 
 	var rs string
 	switch req.GetRequestedState() {
-	case api.SdkCloudBackupRequestedState_SdkCloudBackupRequestedStatePause:
-		rs = api.CloudBackupRequestedStatePause
-	case api.SdkCloudBackupRequestedState_SdkCloudBackupRequestedStateResume:
-		rs = api.CloudBackupRequestedStateResume
+	/*
```

Comment:
Can you add comments to why those lines are commented out? That way the next person reading the code will know why.

Reply:
Done.